### PR TITLE
Add postalCode field support

### PR DESCRIPTION
**Pull request by alex-deaconu**
Enable address validation
User observer for fullAddress value
Send error event if address not valid

### CODE REVIEW

**Comment by alex-deaconu** [comment]
I tested using observers for the individual address fields, but updating the object itself with the same values still triggered the observer. It seems watching the String result works as expected.

@ezequielc please review. Thanks!

File: src/rise-data-weather.js [hunk 1]
```diff
@@ -191,9 +192,19 @@ class RiseDataWeather extends CacheMixin( PolymerElement ) {
...
     RisePlayerConfiguration.DisplayData.onDisplayAddress(( displayAddress ) => {
       this._setDisplayAddress( displayAddress );
+    });
+  }
 
+  _fullAddressUpdated() {
+    var message = "displayAddress is incomplete or missing";
+
+    if ( this.fullAddress ) {
       this._refresh( 0 );
```
Comment (ezequielc) [inline src/rise-data-weather.js]: @alex-deaconu Given the URL changed, we should probably add some code here to deal with a potential refresh in progress. Maybe just by setting _weatherRequestRetryCount to 0. Thanks

Comment (alex-deaconu) [reply src/rise-data-weather.js]: @ezequielc good point. Updated. Let me know if this is what you were referring to. Thanks!

Comment (ezequielc) [reply src/rise-data-weather.js]: @alex-deaconu Yes, that is what I was referring to. Thanks

**Pull request merged**